napari_tiff/napari_tiff_reader.py

```diff
@@ -72,14 +70,30 @@ def zip_reader(path: PathLike) -> List[LayerData]:
 
 
 def tifffile_reader(tif):
-    """Return napari LayerData from largest image series in TIFF file."""
+    """Return napari LayerData from image series in TIFF file."""
+    nlevels = len(tif.series[0])
+    if nlevels > 1:
+        data = [da.from_zarr(tif.aszarr(level=level)) for level in range(nlevels)]
+    else:
+        data = da.from_zarr(tif.aszarr())
```

Comment:
I'm not entirely sure how I feel about making both dask and zarr hard requirements for opening any tiff, even the non-pyramidal ones

Reply:
This is what allows lazy loading & pyramid sizes.
Although this works regardless, I'll split it up and put back asarray() for single series.